From 3f1e499e3786b85b54744a18b5010bfcdaea6c00 Mon Sep 17 00:00:00 2001
From: Bai Ping <ping.bai@nxp.com>
Date: Tue, 13 Dec 2016 16:51:59 +0800
Subject: [PATCH 1363/5242] MLK-13601-01 ARM: dts: imx: update the setpoint
 for imx6ull

commit  d20486159facd43b5f9ebe0c38999a4effdc6d45 from
https://source.codeaurora.org/external/imx/linux-imx.git

According to the latest datasheet(Rev.0,09/2016), two new setpoints
are added on i.MX6ULL, 996MHz/1.25V and 792MHz/1.2V. these two setpoints
can only be enabled when LDO is enabled. In order to cover IR drop and
board tolerance, we add a 25mV margin to the voltage.

Signed-off-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-ldo.dts |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-ldo.dts b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-ldo.dts
index e0dc29b..735b2ef 100644
--- a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-ldo.dts
+++ b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-ldo.dts
@@ -11,12 +11,16 @@
 &cpu0 {
 	operating-points = <
 		/* kHz	uV */
+		996000	1275000
+		792000	1225000
 		528000	1175000
 		396000	1025000
 		198000	950000
 	>;
 	fsl,soc-operating-points = <
 		/* KHz	uV */
+		996000	1175000
+		792000	1175000
 		528000	1175000
 		396000	1175000
 		198000	1175000
-- 
1.7.9.5

